lib/events.js

```diff
@@ -480,3 +484,13 @@ function unwrapListeners(arr) {
   }
   return ret;
 }
+
+function validateEventName(name, paramName) {
+  if (name === undefined) {
+    const { ERR_INVALID_ARG_TYPE } = lazyErrors();
+    const err = new ERR_INVALID_ARG_TYPE(paramName, ['string', 'symbol'], name);
```

Comment:
are there other possible paramNames? Otherwise I think it would be best to inline that string.

Reply:
I agree.